lib/libaccess/aclscan.l

```diff
@@ -64,13 +64,19 @@ variable	[\*a-zA-Z0-9\.\-\_][\*a-zA-Z0-9\.\-\_]*
 
 <<EOF>>		{	
 			yylval.string = NULL;
-			last_string = yylval.string;
+			if (last_string) {
+				PERM_FREE(last_string);
+				last_string = NULL;
+			}
 			return(0);
 		}
 
 {qstring}	{ 
+			if (last_string) {
+				PERM_FREE(last_string);
+				last_string = NULL;
+			}
 			yylval.string = PERM_STRDUP( yytext+1 );
-			last_string = yylval.string;
```

Comment:
Should we really remove that line ?
Is it normal that laststring is now NULL instead of PERM_STRDUP( yytext+1 ); in the old code ?